int32 classid
float32 confidence
int32 xmin
int32 ymin
int32 xmax
int32 ymax
